dev/AppLifecycle/ActivationRegistrationManager.h

```diff
@@ -6,18 +6,37 @@
 
 namespace winrt::Microsoft::ProjectReunion::implementation
 {
+    // Registration constant values.
+    static const std::wstring c_argumentPrefix = L"----";
```

Comment:
>c_argumentPrefix = L"----"; [](start = 30, length = 27)

...c_argumentPrefix{ L"-----" };

ES.23: Prefer the {}-initializer syntax
[https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#es23-prefer-the--initializer-syntax](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#es23-prefer-the--initializer-syntax)

Same comment applies throughout the (not just this file) i.e. never use = for initialization, always use {}